src/Dfc.CourseDirectory.Core/DataStore/Sql/Queries/GetAllRegions.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+using Dfc.CourseDirectory.Core.Models;
+
+namespace Dfc.CourseDirectory.Core.DataStore.Sql.Queries
+{
+    public class GetAllRegions : ISqlQuery<IReadOnlyCollection<Region>>
```

Comment:
There's inconsistency with naming of "GetAll" queries, I'd prefer to see just `Get{Entity}`, without a conditional suffix is enough to imply that it's "All" and this way they sort alphabetically.